zap_text_encoder.go

```diff
@@ -600,9 +601,33 @@ func (enc *textEncoder) tryAddRuneError(r rune, size int) bool {
 }
 
 func (enc *textEncoder) addFields(fields []zapcore.Field) {
-	for i := range fields {
+	for _, f := range fields {
+		if f.Type == zapcore.ErrorType {
+			// handle ErrorType in pingcap/log to fix "[key=?,keyVerbose=?]" problem.
+			// see more detail at https://github.com/pingcap/log/pull/5
+			enc.encodeError(f)
+			continue
+		}
 		enc.beginQuoteFiled()
-		fields[i].AddTo(enc)
+		f.AddTo(enc)
 		enc.endQuoteFiled()
 	}
 }
+
+func (enc *textEncoder) encodeError(f zapcore.Field) {
```

Comment:
Can we make the following modification?
1. s/ err error/ f zapcore.Field
2. line 621: enc.AddString("err", basic)
3. line 629: enc.AddString("errVerbose", verbose)

Reply:
just as morning's discuss, let's use 'error' as key, so we can use current code